tasks/nunjucks.js

```diff
@@ -35,7 +35,7 @@ module.exports = function(grunt) {
 
         async.each(this.files, function(f, done) {
             var filepath = path.join(process.cwd(), f.src[0]);
-            var data = options.data || {};
+            var data = JSON.parse(JSON.stringify(options.data || {}));
```

Comment:
Yep, you're right. Done https://github.com/vitkarpov/grunt-nunjucks-2-html/commit/116867e8a00188fb0d3eb5cc72d7f5dd0bc09c4a